sdk/iot/Azure.Iot.Hub.Service/tests/Azure.Iot.Hub.Service.Tests.csproj

```diff
@@ -6,14 +6,28 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Identity" />
-    <PackageReference Include="nunit" />
-    <PackageReference Include="NUnit3TestAdapter" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" />
-    <PackageReference Include="Moq" />
+    <PackageReference Include="FluentAssertions" />
   </ItemGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\..\..\core\Azure.Core.TestFramework\src\Azure.Core.TestFramework.csproj" />
     <ProjectReference Include="..\src\Azure.Iot.Hub.Service.csproj" />
   </ItemGroup>
+
+  <ItemGroup>
+    <None Update="config\**">
```

Comment:
Is the .gitignore updated to ignore user config files?

Reply:
it is

Reply:
Not by me, i'll check and add it if it is missing.

Reply:
they are.